tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -1264,6 +1264,112 @@ def test_vpc_config(
             "delete_vpcconfig_get_function_response", delete_vpcconfig_get_function_response
         )
 
+    @markers.aws.validated
+    def test_invalid_vpc_config_subnet(
+        self, create_lambda_function, lambda_su_role, snapshot, aws_client, cleanups
```

Comment:
nit: create_lambda_function fixture unused

Reply:
Oh no I'm so sorry, I thought I had push the changes but I forgot! 😭 there was `create_lambda_function` and `cleanups` being unused...

Reply:
No worries at all. It was just a tiny nit ☺️